app/services/collators/gross_income_collator.rb

```diff
@@ -33,6 +37,7 @@ def populate_income_attrs(attrs, category)
 
     def default_attrs
       {
+        # this looks wrong, monthly_student_loan is used for total gross income
```

Comment:
is there a separate ticket to look at this?

Reply:
It does look a bit odd, but in fact is correct.  It's setting up the initial values of `attrs` here, and then adds in for each category on line 25.  I've changed Will's comment to read `# setup initial values here`